packages/protocol/contracts/common/interfaces/ICeloToken.sol

```diff
@@ -9,7 +9,10 @@ interface ICeloToken {
   function initialize(address) external;
   function transferWithComment(address, uint256, string calldata) external returns (bool);
   function burn(uint256 value) external returns (bool);
+  function mint(address to, uint256 value) external returns (bool);
   function name() external view returns (string memory);
   function symbol() external view returns (string memory);
   function decimals() external view returns (uint8);
+  function allocatedSupply() external view returns (uint256);
+  function totalSupply() external view returns (uint256);
```

Comment:
There's an argument that tis should come from the IERC20, but to really a big deal.